testsuite/integration-arquillian/tests/other/console/src/main/java/org/keycloak/testsuite/console/page/clients/clientscopes/ClientScopesEvaluateForm.java

```diff
@@ -81,6 +82,10 @@
     @FindBy(tagName = "textarea")
     private WebElement accessTokenTextArea;
 
+    @FindBy(id = "s2id_users")
+    private ClientSelectModal clientsInput;
```

Comment:
@saibot94 Thanks for the update! Even though it might be a bit confusing that we're using `ClientSelectModal` Page Fragment to select users, I'm gonna approve. ;) Thank you again for your contribution!